deployment_scripts/puppet/modules/plugin_k8s/manifests/calico_node.pp

```diff
@@ -32,6 +32,15 @@
     confdir      => $::plugin_k8s::params::network_plugin_dir,
   }
 
+  # Kubernetes needs nsenter in order to find out container's IP, nsenter is
+  # a part of util-linux starting from version 2.24, so Ubuntu 14.10 and
+  # newer are OK
+  if $::operatingsystem == 'ubuntu' and $::operatingsystemrelease == '14.04' {
+    package {'nsenter':
```

Comment:
We probably should just install this as I've noticed warning about this in the kubelet logs even with flannel.